run-tests.bat

```diff
@@ -4,4 +4,4 @@ virtualenv R:\.venv
 R:\.venv\Scripts\pip install -e . --upgrade --upgrade-strategy=only-if-needed
 R:\.venv\Scripts\pipenv install --dev
 
-SET RAM_DISK=R:&& SET PYPI_VENDOR_DIR=".\tests\pypi\" && R:\.venv\Scripts\pipenv run pytest -n auto -v tests --tap-stream > report.tap

```

Comment:
The `PYPI_VENDOR_DIR` setting is 100% necessary if you want your tests to pass, I'm not sure why you're removing it??

Reply:
This line handles it when running with just `pytest`. https://github.com/pypa/pipenv/blob/master/tests/integration/conftest.py#L71

Reply:
Ah, indeed haha, I must've been running it from the `test` directory and the implicit `./pypa` in `app.py` worked. I had a different version with more explicitly loaded packages that I must've been thinking of. I'll double test that locally and push.